src/features/pastelStatistics/components/chart/LineChart.module.css

```diff
@@ -0,0 +1,99 @@
+.container {
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+  padding: 0 50px;
+  text-align: center;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.lineChartWrap {
+  width: 100%;
+  height: auto;
+  margin-bottom: 20px;
+}
+
+.lineChartContainer {
+  display: block;
+  width: 100%;
+  height: 100%;
+  background-color: #212121;
+}
+
+.lineChartTitle {
+  width: 100%;
+  text-align: left;
+  font-size: 30px;
+}
+
+.lineChartHeader {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  padding-top: 30px;
+  padding-bottom: 10px;
+}
+
+.uploadButton {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 185px;
+  background-color: #c3921f;
+  color: #000;
+  text-align: center;
+  font-weight: bold;
+  font-size: 14px;
+  border-radius: 4px;
+  border-color: #c3921f;
+  margin-left: 8px;
+  margin-right: 8px;
+}
+
+.uploadButton {
```

Comment:
Duplicate `uploadButton` class.

Reply:
thank you for your comment.
- missed [:hover]